### PR TITLE
rename commit to apply

### LINKED ISSUE #759: rename commit to apply
Changes made should be "applied" not "committed".

### PR DESCRIPTION
**Pull request by scholarsmate**
closes #759

### CODE REVIEW

File: src/dataEditor/dataEditorClient.ts [hunk 1]
```diff
@@ -385,7 +385,7 @@ export class DataEditorClient implements vscode.Disposable {
         }
         break
 
-      case MessageCommand.commit:
+      case MessageCommand.apply:
```
Comment (stricklandrbls) [inline src/dataEditor/dataEditorClient.ts:388]: Given that we're going to change the term, I think "Apply" would suffice for the button label but maybe we should make the source code a little more verbose?

`MessageCommand.applyEdits` or `applyChangesToViewport`

I just wonder if we'll end up having to change the term in the source if we choose a generic `apply`.

Comment (scholarsmate) [reply src/dataEditor/dataEditorClient.ts:388]: It's not just applying edits to the viewport, but to the edit model in Ωedit.

Comment (scholarsmate) [reply src/dataEditor/dataEditorClient.ts]: Your requested change has been made to the message enum.

**Review by shanedell — APPROVED** [review]
+1

**Pull request merged**